test/amoc_api_scenarios_handler_SUITE.erl

```diff
@@ -126,40 +121,31 @@ get_url() ->
 
 -spec get_request(string()) -> 
     {integer(), jiffy:jiffy_decode_result()}.
-get_request(URL) ->
-    Header = [],
-    HTTPOpts = [],
-    Opts = [],
-    Result = httpc:request(get,
-                           {URL, Header},
-                           HTTPOpts,
-                           Opts),
-
-    {ok, {{_HttpVsn, CodeHttp, _Status}, _, Body}} = Result,
-    BodyErl = case Body of
-                  [] -> 
-                      [];
-                  _ ->
-                      jiffy:decode(erlang:list_to_bitstring(Body))
-              end,
-    {CodeHttp, BodyErl}.
+get_request(Path) ->
+    request(get, Path).
 
 -spec post_request(string(), string()) ->
     {integer(), jiffy:jiffy_decode_result()}.
-post_request(URL, RequestBody) ->
-    Header = "",
-    Type = "application/json",
-    HTTPOpts = [],
-    Opts = [],
-    Result = httpc:request(post,
-                           {URL, Header, Type, RequestBody},
-                           HTTPOpts,
-                           Opts),
-    {ok, {{_HttpVsn, CodeHttp, _Status},_, Body}} = Result, 
+post_request(Path, RequestBody) ->
```

Comment:
As the requests are used in different test modules they should be extracted into a separate module with test utilities.